recipes/thriftpy/meta.yaml

```diff
@@ -0,0 +1,45 @@
+package:
+  name: thriftpy
+  version: "0.3.5"
+
+source:
+  git_url: https://github.com/eleme/thriftpy
+  git_rev: v0.3.5
```

Comment:
Again, could we please use the [PyPI release]( https://pypi.python.org/pypi/thriftpy/0.3.7 ). Also, there appears to be `0.3.7`, which is newer. Would this work for you?